msg/estimator_status.msg

```diff
@@ -106,12 +106,10 @@ uint16 solution_status_flags # Bitmask indicating which filter kinematic state o
 
 float32 time_slip # cumulative amount of time in seconds that the EKF inertial calculation has slipped relative to system time
 
-uint8 pre_flt_fail_flags # Bitmask indicating which innovation preflight check failed
-# 0 - True if the heading innovation check failed
-# 1 - True if the horizontal velocity innovation check failed
-# 2 - True if the vertival velocity innovation check failed
-# 3 - True if the height innovation check failed
-
+bool pre_flt_fail_innov_heading
```

Comment:
@dagar better?

Reply:
Thank you, much appreciated. I'll take a pass at creating an `estimator_status_flags` one of these days.

Reply:
Note for later, we may want to consider peeling this off into a separate message and then only run the code (and publish the message) during preflight.